docs/agent/flag-ec2-use-windows-prefix-detection.md

```diff
@@ -0,0 +1,21 @@
+# Windows EC2 hostname starting with `EC2AMAZ-`
```

Comment:
We should create this page in the documentation repo under Agent FAQ. Please don't create it here. We spent a lot of time migrating agent docs.

Reply:
@ruthnaebeck 
What is the full link we can use?
Please note that we cannot change the link targeted by `https://dtdg.co/ec2-use-win-prefix-detection`. It means the link must be valid and **cannot be changed later** as it is not possible for us to update the link on existing installation of the agent.

Reply:
Are we already using `https://dtdg.co/ec2-use-win-prefix-detection` somewhere? If not, lets not use `dtdg.co` links for Agent install files since they cannot be changed.

We would put the doc here:
`https://github.com/DataDog/documentation/tree/master/content/en/agent/faq`
and the web URL could be (last part can be renamed if needed):
`https://docs.datadoghq.com/agent/faq/agent/flag-ec2-use-windows-prefix-detection`

Reply:
> Are we already using https://dtdg.co/ec2-use-win-prefix-detection somewhere? If not, lets not use dtdg.co links for Agent install files since they cannot be changed.
We are not using it for now. What do you recommend as an alternative to `dtdg.co links`?

Reply:
I recommend we use `docs.datadoghq.com` links. If there is a concern about having a short URL, we can create a short alias for a page, for example:
`https://docs.datadoghq.com/agent/faq/flag-ec2-use-windows-prefix-detection` could be:
`https://docs.datadoghq.com/<anything_here>`